images/dockerfiles/horizon-telegraf/telegraf.conf

```diff
@@ -0,0 +1,116 @@
+# Telegraf configuration for horizon:
+# this configuration forwards statsd metrices received from horizon's nginx to a collection point. The output is formatted with influx-db metric format and sent to http://metrics.kininfrastructure.com:8086. This configuration does not expect any parameters. You can tweak the percentile generated by the statsd-input with the 'percentiles' key below.
+
+# Global tags can be specified here in key="value" format.
```

Comment:
i prefer removing all comments and also all default configs - i can go to the config reference on my own if needed. this will slow us down when i need to scan through the configuration fast